src/main/resources/templates/newSurvey.html

```diff
@@ -37,11 +39,9 @@ <h2 th:text="'Survey Name: '+${survey.getName()}">Survey Name</h2>
             <td th:text="${question.getId()}"></td>
             <td><a th:href="@{/question/{q}(q=${question.getId()})}" th:text="${question.getPrompt()}"></a></td>
             <td>
-                <form method="post" th:action="@{/question/delete}">
```

Comment:
Nice job, it's good we're getting rid of this janky code

Reply:
htmx is pretty nice